src/ceph-disk

```diff
@@ -2435,7 +2435,10 @@ def get_partition_type(part):
     if 'blkid' not in warned_about:
         LOG.warning('Old blkid does not support ID_PART_ENTRY_* fields, trying sgdisk; may not correctly identify ceph volumes with dmcrypt')
         warned_about['blkid'] = True
-    (base, partnum) = re.match('(\D+)(\d+)', part).group(1, 2)
```

Comment:
the **split_dev_base_partnum** should be used to avoid duplicating this detection logic